riff-raff.yaml

```diff
@@ -20,3 +22,23 @@ deployments:
       cloudFormationStackName: TagManager
       cloudFormationStackByTags: false
       templatePath: tag-manager.yaml
+
+  cloudformation-pillar-db:
+    type: cloud-formation
+    app: tag-manager-pillar-db
+    parameters:
+      templatePath: pillar-dynamo.yaml
+      prependStackToCloudFormationStackName: false
+      cloudFormationStackName: pillars-dynamo
+      cloudFormationStackByTags: false
```

Comment:
Generally I prefer tag based discovery, but this repo is already using stack name based discovery and I think it's better to remain consistent within the same repo

Reply:
Fair enough